docker/db/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM jpetazzo/pgsql
```

Comment:
There are many postgres images.  If there is some 'official' ones from the project maintainers ...  This may be even be one of those.